src/deroff.rs

```diff
@@ -555,6 +700,20 @@ impl Deroffer {
     fn flush_output<W: std::io::Write>(&mut self, mut write: W) {
         write.flush().unwrap()
     }
+
+    fn do_line(&self, s: &str) -> bool {
+        match s.chars().nth(0) {
+            Some('.') | Some('\'') => !request_or_macro(s),
+            Some(c) => {
+                if self.tbl {
+                    do_tbl(s)
+                } else {
+                    text(s)
+                }
+            }
+            None => unreachable!(),
```

Comment:
Nit: Not locally unreachable. Maybe
```suggestion
            None => panic!("do_line` called with empty string as argument"),
```
instead?